app/utils.py

```diff
@@ -111,7 +111,7 @@ def strip_whitespace_from_data(data):
         if isinstance(value, list):
             # Strip whitespace and remove empty items from lists
             data[key] = list(
-                filter(lambda x: x not in ['', None],
+                filter(lambda x: x != '',
```

Comment:
This seems okay to me.  As it's used for `list`s and `boolean_list`s (and empty list fields are sent back as `''`, right?), it looks like a fairly small change.